tests/onegov/town6/test_views_resource_recipients.py

```diff
@@ -0,0 +1,38 @@
+import transaction
```

Comment:
Should we also add this test to org?

Reply:
The resources page is only used on town6 instances as far as I know, but I could add the test for the case that other instances will use it too in the future.